.gitignore

```diff
@@ -3,6 +3,7 @@ yarn.lock
 yarn-error.log
 .cache
 /public
+/external
```

Comment:
you can delete this folder locally, no need to ignore it. It's legacy when we used git submodules which was removed some weeks ago